packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -243,6 +243,9 @@ async function moveTeam({
       language: "en",
       inviterName: null,
       teamId: org.id,
+      // This is important so that if we re-invite existing users accidentally, we don't endup erroring out.
+      // Because this is a bulk action that could be taken from organization payment webhook, we could have cases where a user was just invited through another team migration in parallel.
+      isDirectUserAction: false,
```

Comment:
We don't want to crash on user errors in the migration flow as that is triggered asyncronously through webhook. We just ignore those errors.

This is what automatically happens when we invite more than 1 user